File: Elice/Day03/caboooom_240710/Main.java
package Elice.Day03.caboooom_240710;

import java.io.BufferedReader;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        String input = reader.readLine();

        StringBuilder builder = new StringBuilder();
        char[] inputArr = input.toCharArray();

        for(int i=0; i<inputArr.length; i++) {
            if(Character.isDigit(inputArr[i]) && Character.isDigit(inputArr[i+1])) {
                builder.append("1").append("+");
            } else if(Character.isDigit(inputArr[i]) && inputArr[i+1] == '(') {
                builder.append(inputArr[i]).append("*");
            } else if(Character.isDigit(inputArr[i]) && inputArr[i+1] == ')') {
                builder.append("1");
            } else{
                builder.append(inputArr[i]);
            }
        }

        System.out.println(builder);


    }
}

File: Elice/Day03/caboooom_240710/main.py
import sys

input = input()

builder = []
inputArr = list(input)

for i in range(len(inputArr)):
    if i < len(inputArr) - 1:
        if inputArr[i].isdigit() and inputArr[i + 1].isdigit():
            builder.append("1+")
        elif inputArr[i].isdigit() and inputArr[i + 1] == '(':
            builder.append(inputArr[i] + "*")
        elif inputArr[i].isdigit() and inputArr[i + 1] == ')':
            builder.append("1")
        else:
            builder.append(inputArr[i])
    else:
            builder.append(inputArr[i])


for i in range(len(builder) - 1):
    if builder[i].isdigit() and builder[i + 1] == '(':
        builder[i] += '*'

print(eval("".join(builder)))